### PR TITLE
add options.exclude

### PR DESCRIPTION
**Pull request by CreaturesInUnitards**
fixes the only thing missing from my now-very-@PepsRyuu-centric toolchain :)

### CODE REVIEW

**Comment by PepsRyuu** [comment]
Hahahaha. Thanks for the PR, will give it a try over the weekend. :)

**Comment by PepsRyuu** [comment]
This doesn't work with relative paths that begin with "./", at least not for me anyways. When I tested it, ```file``` gives ```public/subdir/somefile.txt``` rather than ```./public/subdir/somefile.txt```. Being a ```RegExp``` could also introduce some issues with wildcards symbols developers are used to such as ```*/**```. While the ideal solution would be to use a proper matching library, to keep the solution simple for the time being, what needs to be done is to resolve ```file``` into a relative path to ```process.cwd()```, and then use ```.startsWith()``` instead.

**Comment by CreaturesInUnitards** [comment]
yeah I was trying to match the relative naïveté of the rest, but you're totally right — I'll hit that in a couple of hours, thanks

**Comment by CreaturesInUnitards** [comment]
@PepsRyuu as you suggested, this commit normalizes both `includes` and `excludes` relative to `process.cwd()`, and filters via `!includeFile.startsWith(excludeFile)`

File: index.js [hunk 1]
```diff
@@ -4,63 +4,74 @@ let klawSync = require('klaw-sync');
...
-    }
+	for (let file in bundle) {
+		if (pattern.test(file)) {
+			return file;
+		}
+	}
+}
+
+function normalizedFile(file) {
+	return `${process.cwd()}/${file.match(/[^\.\/](.+)/g)}`
```
Comment (CreaturesInUnitards) [inline index.js]: This ignores leading `.` and/or `/` with extreme prejudice. The thinking being that any `exclude` must necessarily reside within an `include`, so strict normalization is the sanest approach.

**Comment by PepsRyuu** [comment]
Just two things:

* Doesn't work on Windows. Instead of the regex, easier to use Node's built-in path resolver: ```return path.resolve(process.cwd(), file);```
* The spaces got switched to tabs, makes it tricky to see what's changed. 

Otherwise it works well!

**Comment by CreaturesInUnitards** [comment]
"Doesn't work on Windows" is a strong candidate to become my next tattoo 😂

**Comment by PepsRyuu** [comment]
Released in ```0.2.0``` :)

**Comment by CreaturesInUnitards** [comment]
Outstanding, thanks! Just FYI our new nollup-based toolchain is near-perfect — production JS is about 1.3M, but per-change dev wait time is almost imperceptible. Thanks again for the really excellent work.

**Pull request merged**